client/src/components/helpers/simpletons.js

```diff
@@ -1,7 +1,7 @@
-const trim = val => val.trim();
-
-const truncate = (str, length = 10) => {
-  return str.length > length ? `${str.substr(0, length)}...` : str;
-};
-
-export { trim, truncate };
+const trim = val => val.trim();

```

Comment:
Why use an internal method name as a function name? Look for a proper name. Besides what's the point of that function?

Reply:
I will remove it immediately